ambry-network/src/main/java/com/github/ambry/network/http2/Http2ChannelPoolHandler.java

```diff
@@ -64,5 +72,19 @@ public void channelCreated(Channel ch) throws Exception {
         .build());
     pipeline.addLast(new Http2MultiplexHandler(new ChannelInboundHandlerAdapter()));
   }
+
+  @ChannelHandler.Sharable
+  private static class ConnectionInboundExceptionHandler extends ChannelInboundHandlerAdapter {
```

Comment:
what is this? This handler is not used anywhere, and it seems like we are trying to print out some info when it's a stream exception, which makes this a stream handler, doesn't stream handler belong to Http2NetworkClient.StreamChannelInitializer?

Reply:
If a request is drop, its streamChannel will be closed, but if a response is already inflight, and then arrives client. 
Http2MultiplexHandler will report unknown stream id error and print a long stack. 
The handler here is to avoid such useless stack in our log.

Reply:
Are we sure that all resources on child channels will be released if the exception is caught and swallowed here?